### DISCUSSION TITLE
AppSelfHost wrong X-Powered-By

### DISCUSSION THREAD

**Discussion by c0nnex** [ServiceStack/Discuss#40] [General]
- ServiceStack 6.4.0
- Selfhosted App derived from AppSelfHostBase
- in Configure first line is : 
     Licensing.RegisterLicense(ssLic);
- License is validated ok
- "X-Powered-By:" header shows "ServiceStack/6.40 NETFX/Windows/netfx/NO"

Most likely caused by Env static Contructor. License is not yet set by the time it's contructed. Should be updated when license changes.

**Comment by mythz** [comment]
This should be resolved from [this commit](https://github.com/ServiceStack/ServiceStack/commit/c7f4d7c483327149a7f95d24bfe1554378ba9dd0). This change is available from v6.4.1+ that's now [available on MyGet](https://docs.servicestack.net/myget).

Thx for reporting!

**Reply by c0nnex** [reply]
Thanks for the quick fix. Since ServiceStack is used in production environments i'll wait for next offical release.
